debug/utils.py

```diff
@@ -56,6 +56,10 @@ def dump(self):
     def display_hint(self):
         return 'map'
 
+    @property
+    def address(self):
+        return str(self._obj.address)
```

Comment:
Natural type for `self._obj.address` is integer. Please do not cast it to string by default.

Reply:
Corrected in cf3bf8d.